db.js

```diff
@@ -0,0 +1,519 @@
+const pg = require('pg');
+
```

Comment:
We need to break this apart. Seeding development data, should be its own file entirely, sometimes multiple files depending on how the data is generated (faker vs you writing it yourself), but also the models should each have their own file. This file is getting enormous and will cause lots of git conflicts as multiple people have to work in it at the same time.